code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -595,3 +595,45 @@
 
 /obj/effect/mob_spawn/human/pirate/gunner
 	rank = "Gunner"
+
+/obj/effect/mob_spawn/human/ghostcafe
+	name = "ghost cafe sleeper"
+	uses = -1
```

Comment:
```suggestion
	permanent = TRUE
```

Reply:
the "permanent" var only affects whether the spawner will be qdel'd after running out of uses
here it has no effect, because the spawner will never run out of uses
```
	if(!permanent && !uses)
		qdel(src)
```

Reply:
ok.